custom/auth/ante/burntax.go

```diff
@@ -5,12 +5,14 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	"github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 )
 
 // TaxPowerUpgradeHeight is when taxes are allowed to go into effect
 // This will still need a parameter change proposal, but can be activated
 // anytime after this height
 const TaxPowerUpgradeHeight = 9346889
+const WhitelistHeight = 12345678910
```

Comment:
Don't we have to specify the real height of this?

Reply:
I think it is up to @edk208 to decide

Reply:
Yes we are still waiting on @edk208 to provide a blockheight for whitelist activation :)

Reply:
do we need a height guard?  wouldn't it just kick in at the time of the upgrade if we use a software upgrade handler?

Reply:
yep, it will just kick in at the time of software upgrade